content/providers/quick-start.json

```diff
@@ -19,21 +19,7 @@
     },
     "codeExamples": [
       {
-        "name": "Transpiling",
-        "runMethod": "backend",
-        "fullCode": [
-          "# Build circuit",
-          "from qiskit.circuit.library import QuantumVolume",
-          "circuit = QuantumVolume(5)",
-          "&nbsp;",
-          "# Transpile circuit",
-          "from qiskit import transpile",
-          "transpiled_circuit = transpile(circuit, basis_gates=['sx', 'rz', 'cx'])",
-          "transpiled_circuit.draw()"
-        ]
-      },
-      {
-        "name": "Sampling Bell Circuit",
+        "name": "Sample a Bell State",
```

Comment:
for all tabs, `Transpiler` should go first

Reply:
Resolved in https://github.com/Qiskit/qiskit.org/pull/3113/commits/6bc0dbf465f3d04bb6012c1df1a78f1b6e8ca31e